components/style/gecko_conversions.rs

```diff
@@ -0,0 +1,32 @@
+//! This module contains conversion helpers between Servo and Gecko types
+//! Ideally, it would be in geckolib itself, but coherence
+//! forces us to keep the traits and implementations here
```

Comment:
I think I would prefer a custom trait if it means that we can keep this logic in geckolib.